### ISSUE TITLE
e.tags.forEach is not a function

### ISSUE THREAD

**Issue by xdfwsl** [cwxyz007/vscode-hexo-utils#20]
When I save the file the hexo utils always pops up the message 'e.tags.forEach is not a function' and even the file is  just a empty one.

**Comment by xdfwsl** [comment]
![微信图片_20191015165541](https://user-images.githubusercontent.com/10320275/66816484-b8f3c280-ef6c-11e9-9138-784648733b3e.png)

**Comment by cwxyz007** [comment]
This could be is a bug and thanks for feedback.

For now, please try this

```md
---
title: test1
date: 2019-10-15T08:32:37.226Z
tags: []
---
```

**Comment by xdfwsl** [comment]
![微信图片_20191015165541](https://user-images.githubusercontent.com/10320275/66817018-9f9f4600-ef6d-11e9-9b57-1f71bcfb0bee.png)


The problem is not solved.

**Comment by cwxyz007** [comment]
I just tested it, it works fine for me. 

Please try reinstall this extension, if it not work, please upload the file you created, I will test it later.

**Comment by xdfwsl** [comment]
[test1.zip](https://github.com/cwxyz007/vscode-hexo-utils/files/3728399/test1.zip)

**Comment by xdfwsl** [comment]
I have reinstalled this extension yet, but it's doesn't work.

**Comment by cwxyz007** [comment]
I have tested the file you upload and it works fine.
So, It will very helpful to location the bug If you provide these information.

```
vscode-hexo-utils version:
system os:
vscode version:
file encoding: 
```
![image](https://user-images.githubusercontent.com/14226737/66886589-74b6ff80-f00a-11e9-9030-df86fe646739.png)

**Comment by cwxyz007** [comment]
Closed: 5e30f90d8eb43ef9413a56addb5e7f96aecefd8f

**Comment by xdfwsl** [comment]
it works
